torch/csrc/jit/passes/shape_analysis.cpp

```diff
@@ -47,7 +47,7 @@ IValue representativeValue(Value* v) {
   if (CompleteTensorTypePtr type = type_->cast<CompleteTensorType>()) {
     auto backend = type->device() == -1 ? at::Backend::CPU : at::Backend::CUDA;
     at::DeviceGuard device_guard(type->device());
-    auto& attype = at::getType(backend, type->scalarType());
+    auto& attype = at::getNonVariableType(backend, type->scalarType());
```

Comment:
This should use options, but we don't have a tensor constructor that also takes strides.